ledger-core/v2/application/bootstrap/keys.go

```diff
@@ -89,6 +90,10 @@ func createKeysInDir(
 			return nil, errors.Wrap(err, "[ createKeysInDir ] couldn't marshal keys")
 		}
 
+		if properNames {
+			keyname = n.CertName
+		}
```

Comment:
Is this flag really needed? Why not to use keyname = n.CertName always?

Reply:
Not really, it's only for backward compat with launchnet.sh. Don't know if it really want to invest time in it now